doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -891,6 +903,39 @@ To delete a client, follow these steps.
 2.  In the window that opens, click **DELETE** and then click **YES**. If there are no users for the signature key nor for the certificate associated then an option is presented to delete the client's certificates. To delete the certificates, click **YES** again.
 
 
+### 4.7 Disabling Client Subsystem Temporarily
+
+Security Server client subsystem in "Registered" state may be disabled temporarily for maintenance purposes. When subsystem is disabled it cannot use or provide X-Road services.
+
+When client subsystem is disabled in Security Server it first enters "Disabling in progress" state. Once the state change is propagated through the global configuration, client subsystem enters the "Disabled" state.
+
+Client subsystem in "Disabled" state may be enabled again. When it is enabled in the Security Server it first enters "Enabling in progress" state. Once state change is propagated, "Registered" state is restored.
```

Comment:
Similarly, it would be good to explain when the subsystem becomes enabled - does it happen already in the "Registration in progress" state or when it reaches the "Registered" state?